skins/oasis/modules/LatestActivityController.class.php

```diff
@@ -58,6 +60,14 @@ public function executeIndex() {
 		$this->setVal( 'activityIcon', DesignSystemHelper::renderSvg( 'wds-icons-activity-small', 'wds-icon wds-icon-small wds-activity-icon' ) );
 		$this->setVal( 'renderCommunityEntryPoint', !empty( $wgEnableCommunityPageExt ) );
 
+		// TODO: https://wikia-inc.atlassian.net/browse/CAKE-4746
+		$currentPageTitle = $this->getContext()->getTitle()->getPrefixedText();
+		if ( $wgEnableTriviaQuizzesExt && in_array($currentPageTitle, $wgTriviaQuizzesEnabledPages) ) {
+				$this->setVal( 'renderTriviaQuizzes', $wgEnableTriviaQuizzesExt );
+				$this->setVal( 'moduleHeader', wfMessage('trivia-quizzes-featured-quizzes-header')->escaped() );
+				$this->setVal( 'activityIcon', DesignSystemHelper::renderSvg( 'wds-icons-checkbox-small', 'wds-icon wds-icon-small wds-checkbox-icon' ) );
```

Comment:
Minor: remove an indentation level on these three.

Reply:
done in b28949d